packages/solutions/module-tools/src/locale/en.ts

```diff
@@ -25,4 +35,9 @@ export const EN_LOCALE = {
       lang: 'set new command language(en or zh)',
     },
   },
+  log: {
+    dev: {
+      no_devtools: noDevTools,
```

Comment:
![image](https://user-images.githubusercontent.com/7237365/220032211-c9f8618d-39ba-4c1a-b6b5-eb76a34e4005.png)

This file uses a mix of camelCase and underscore styles, maybe we should unify the naming.

Reply:
update